src/components/LoginForm..jsx

```diff
@@ -0,0 +1,100 @@
+import React, { useState } from 'react';
+import {Tab, Tabs, TextField, Stack, Button, Container } from "@mui/material";
+import { Box } from '@mui/system';
+
+const passwordMatchingText = {
+    error: "Passwords do not match",
+    valid: ""
+}
+function LoginForm() {
+    const [currentTabIndex, setCurrentTabIndex] = useState(0);
+    const [email, setEmail] = useState('');
+    const [password, setPassword] = useState('');
+    const [passwordVerification, setPasswordVerification] = useState('');
+    const [passwordMatchingMsg, setPasswordMatchingMsg] = useState(passwordMatchingText.valid)
+    
+    const handleTabChange = (event, tabIndex) => {
+        setCurrentTabIndex(tabIndex);
+        setEmail('')
+        setPassword('')
+        setPasswordVerification('')
+        setPasswordMatchingMsg(passwordMatchingText.valid)
+    };
+
+    const handleEmailChange = (event) => {
+        setEmail(event.target.value)
+    }
+
+    const handlePasswordChange = (event) => {
+        setPassword(event.target.value)
+        setPasswordMatchingMsg(passwordMatchingText.valid)
+    }
+
+    const handlePasswordVerificationChange = (event) => {
+        setPasswordVerification(event.target.value)
+        setPasswordMatchingMsg(passwordMatchingText.valid)
+    }
+
+    /*
+    TODO:
+    - Handle api status code (200 vs != 200)
+    */
+    const onLogin = (event) => {
+        event.preventDefault()
+        alert("Welcome " + email)
+
+    }
+
+    /*
+    TODO:
+    - Handle api status code (200 vs != 200)
+    */
+    const onCreateAccount = (event) => {
+        event.preventDefault()
+        if (password !== passwordVerification) {
+            setPasswordMatchingMsg(passwordMatchingText.error)
+        }
+    }
+
+    return (
+        <Container sx={{width: "25%",border: "1px solid gray" }}>
+            <Tabs value={currentTabIndex} sx={{ borderBottom: 1}} onChange={handleTabChange}>
```

Comment:
Sometimes there is blank space after the opening curly brace and sometimes there is not. Ideally we should use prettier to reformat the file for us so we don't have to think about it.

Reply:
Hey good catch, I forgot to use a linter on this - this should be added to the project (and ran) 

- https://github.com/secure-password-manager/PasswordManagerWeb/pull/13/commits/34072fe450f3d54a6e5266eb44939440a686822b#